backend/src/test/java/com/ody/meeting/controller/MeetingControllerTest.java

```diff
@@ -166,31 +172,117 @@ void validateInviteCode() {
                 .statusCode(404);
     }
 
-    @DisplayName("참여자들의 위치 현황 목록 성공하면 200응답 반환한다")
-    @Test
-    void findAllMateEtas() {
-        Location origin = Fixture.ORIGIN_LOCATION;
-        Location target = Fixture.TARGET_LOCATION;
-        Member member = memberRepository.save(Fixture.MEMBER1);
-        Meeting meeting = new Meeting("모임1", LocalDate.now(), LocalTime.now(), target, "초대코드1");
-        meetingRepository.save(meeting);
-        Mate mate = mateRepository.save(new Mate(meeting, member, new Nickname("은별"), origin, 10L));
-        etaRepository.save(new Eta(mate, 10L));
+    @DisplayName("Eta API 테스트")
+    @Nested
+    class EtaTest {
 
-        MateEtaRequest mateEtaRequest = new MateEtaRequest(false, origin.getLatitude(), origin.getLongitude());
+        @DisplayName("참여자들의 위치 현황 목록 성공하면 200응답 반환한다")
+        @Test
+        void findAllMateEtas() {
+            Location origin = Fixture.ORIGIN_LOCATION;
+            Location target = Fixture.TARGET_LOCATION;
+            Member member = memberRepository.save(Fixture.MEMBER1);
+            Meeting meeting = new Meeting("모임1", LocalDate.now(), LocalTime.now(), target, "초대코드1");
+            meetingRepository.save(meeting);
+            Mate mate = mateRepository.save(new Mate(meeting, member, new Nickname("은별"), origin, 10L));
+            etaRepository.save(new Eta(mate, 10L));
 
-        RestAssured.given()
-                .log()
-                .all()
-                .header(HttpHeaders.AUTHORIZATION,
-                        "Bearer device-token=" + Fixture.MEMBER1.getDeviceToken().getDeviceToken())
-                .body(mateEtaRequest)
-                .contentType(ContentType.JSON)
-                .when()
-                .patch("/v1/meetings/1/mates/etas")
-                .then()
-                .log()
-                .all()
-                .statusCode(200);
+            MateEtaRequest mateEtaRequest = new MateEtaRequest(false, origin.getLatitude(), origin.getLongitude());
+
+            RestAssured.given()
+                    .log()
+                    .all()
+                    .header(HttpHeaders.AUTHORIZATION,
+                            "Bearer device-token=" + Fixture.MEMBER1.getDeviceToken().getDeviceToken())
+                    .body(mateEtaRequest)
+                    .contentType(ContentType.JSON)
+                    .when()
+                    .patch("/v1/meetings/1/mates/etas")
+                    .then()
+                    .log()
+                    .all()
+                    .statusCode(200);
```

Comment:
[제안]

축약 한 번 부탁드립니다!

```suggestion
            RestAssured.given().log().all()
                    .header(HttpHeaders.AUTHORIZATION,
                            "Bearer device-token=" + Fixture.MEMBER1.getDeviceToken().getDeviceToken())
                    .body(mateEtaRequest)
                    .contentType(ContentType.JSON)
                    .when()
                    .patch("/v1/meetings/1/mates/etas")
                    .then().log().all()
                    .statusCode(200);
```

Reply:
반영했습니다!

Reply:
해당 부분 수정하는 김에 다른 코드들도 한번에 수정했습니다 ~
그런데 개행을 수정해도 변경 사항에 안들어가는거보니 커밋해도 반영안된 것들이 조금씩 있을 것 같네요 🥲